src/components/Accordion.js

```diff
@@ -24,6 +24,38 @@ document.addEventListener(
   false
 );
 
+document.addEventListener(
+  "focusin",
+  onAccordionFocus => {
+    const { target } = onAccordionFocus;
+    changeFocusClass(target, "open");
+  },
+  false
+);
+
+document.addEventListener(
+  "focusout",
+  onAccordionFocus => {
+    const { target } = onAccordionFocus;
+    changeFocusClass(target, "close");
+  },
+  false
+);
+
+const changeFocusClass = (target, status) => {
+  var ua = window.navigator.userAgent;
+  var isIE = /MSIE|Trident|Edge\//.test(ua);
+  if (isIE) {
```

Comment:
This check should around the event listeners, so that the listeners will only run in ie and edge browsers.